tgui/packages/tgui/interfaces/Newscaster.js

```diff
@@ -0,0 +1,632 @@
+          </Box>
+          <Box>
+            <Button
+              icon="newspaper"
+              content="Print Newspaper"
+              disabled={paper <= 0}
+              onClick={() => act('printNewspaper')} />
+          </Box>
+        </Stack.Item>
+      </Stack>
+    </Section>
+  );
+};
+
+/** Channel select is the left-hand menu where all the channels are listed. */
+const NewscasterChannelSelector = (props, context) => {
+  const { act, data } = useBackend(context);
+  const [wantedScreen, setWantedScreen] = useLocalState(context, 'wantedScreen', false);
+  const [channelScreen, setChannelScreen] = useLocalState(context, 'channelScreen', false);
+  const {
+    channels = [],
+    viewing_channel,
+    security_mode,
+    wanted = [],
+  } = data;
+  return (
+    <Section
+      minHeight="100%"
+      width={(window.innerWidth - 410) + "px"}>
+      <Tabs vertical>
+        {wanted.map(activeWanted => (
+          <Tabs.Tab
+            pt={0.75}
+            pb={0.75}
+            mr={1}
+            key={activeWanted.index}
+            icon={activeWanted.active ? "skull-crossbones" : null}
+            textColor={activeWanted.active ? "red" : "grey"}
+            onClick={() => { act('toggleWanted',
+              setWantedScreen(true)); }}>
+            Wanted Issue
+          </Tabs.Tab>
+        ))}
+        {channels.map(channel => (
+          <Tabs.Tab
+            key={channel.index}
+            pt={0.75}
+            pb={0.75}
+            mr={1}
+            selected={viewing_channel === channel.ID}
+            icon={channel.censored ? 'ban' : null}
+            textColor={channel.censored ? "Red" : "white"}
+            onClick={() => act('setChannel', {
+              channel: channel.ID,
+            })}>
+            {channel.name}
+          </Tabs.Tab>
+        ))}
+        <Tabs.Tab
+          pt={0.75}
+          pb={0.75}
+          mr={1}
+          textColor="white"
+          color="Green"
+          onClick={() => act('startCreateChannel', setChannelScreen(true))}>
+          Create Channel [+]
+        </Tabs.Tab>
+      </Tabs>
+    </Section>
+  );
+};
+
+const processedText = value => {
+  const textHtml = {
+    __html: sanitizeText(marked(value, {
+      breaks: true,
+      smartypants: true,
+      smartLists: true,
+      baseUrl: 'thisshouldbreakhttp',
+    })),
+  };
+  return textHtml;
+};
+
+
+/** This is where the channels comments get spangled out (tm) */
+const NewscasterChannelMessages = (_, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    messages = [],
+    viewing_channel,
+    security_mode,
+    channelCensored,
+    channelLocked,
+    channelAuthor,
+    user,
+  } = data;
+  if (channelCensored) {
+    return (
+      <Section color="Red">
+        <b>ATTENTION:</b> Comments cannot be read at this time.<br />
+        Thank you for your understanding, and have a secure day.
+      </Section>
+    );
+  }
+  return (
+    <Section fill scrollable>
```

Comment:
This being scrollable is pointless, because the entire window content is scrollable, making this scrollbar non-functional.

```suggestion
    <Section>
```